CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 ## v4.0.0-beta.1
 
```

Comment:
Won't this be `v4.0.0-beta.2` as `1` is already published

Reply:
Sure. Updated.

Reply:
That should most likely be a new entry, with just what is in your PR listed. Rather than editing beta 1 release notes

Reply:
@davidjbradshaw 
Gotcha.